benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/BenchmarksUtil.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.benchmarks.trace;
+
+import io.opencensus.impllite.trace.TraceComponentImplLite;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+
+/** Util class for Benchmarks. */
+final class BenchmarksUtil {
+  private static final TraceComponentImplLite traceComponentImplLite = new TraceComponentImplLite();
+
+  static Tracer getTracer(String implementation) {
+    if (implementation.equals("impl")) {
+      // We can return the global tracer here because if impl is linked the global tracer will be
+      // the impl one.
+      // TODO(bdrutu): Make everything not be a singleton (disruptor, etc.) and use a new
+      // TraceComponentImpl similar to TraceComponentImplLite.
+      return Tracing.getTracer();
```

Comment:
Should this method check that the tracer is the correct class in this case?

Reply:
we do not have a different Tracer implementation, the only difference is the event-queue used (similar for stats). Once the TODO is resolved this will not be a problem.

Reply:
Right, and we can't access the TraceComponent to check its class.

Reply:
Correct. We will fix this with the TODO.